core/src/main/resources/lib/form/repeatable/repeatable.js

```diff
@@ -166,27 +166,29 @@ Behaviour.specify("DIV.repeated-container", 'repeatable', -100, function(e) {
 });
 
     // button to add a new repeatable block
-Behaviour.specify("INPUT.repeatable-add", 'repeatable', 0, function(e) {
-        makeButton(e,function(e) {
+Behaviour.specify("BUTTON.repeatable-add", 'repeatable', 0, function(e) {
+        e.addEventListener('click', function(e) {
```

Comment:
not sure where exactly the issue is but the 'path' generation isn't working right for hetero elements.

Steps:

1. http://localhost:8080/script
2. `jenkins.formelementpath.FormElementPathPageDecorator.ENABLED = true`
3. Create new freestyle job
4. Add post-build action such as archive artifacts

Check path of post-build artifacts button by inspecting the button:

expected:
`/hetero-list-add[publisher]`

actual:
`/publisher[ArtifactArchiver]/hetero-list-add[publisher]`

This can be reproduced with this test: https://github.com/jenkinsci/acceptance-test-harness/blob/01bd5840273170bbe3c6c7c82aa9479acb5181c1/src/test/java/core/FreestyleJobTest.java#L234

Reply:
Should be fixed now - `hetero-list-add` still uses YUI so needed to update the condition to reflect that :)

Reply:
yeah that fixed it